packages/e2e-tests/tests/focus_mode-01.test.ts

```diff
@@ -34,7 +34,7 @@ test("focus_mode-01: should filter messages as regions based on the active focus
   await setFocusRange(page, { startTimeString: "0:04", endTimeString: "0:05" });
 
   // Verify fewer messages
-  await verifyConsoleMessage(page, "Log point", "log-point", 11);
+  await verifyConsoleMessage(page, "Log point", "log-point", 10);
```

Comment:
I'm not sure why this test had 11 before.

I only see 10 messages in the Console after focusing on 0:04-0:05– both in this branch and [in production](https://app.replay.io/recording/7e0c2f3e-ae52-4af2-8603-6234de4eda10).